packages/components/src/internal/components/entities/actions.ts

```diff
@@ -940,9 +941,26 @@ export const getOriginalParentsFromLineage = async (
         Object.values(originalParents).forEach((parentTypes: List<EntityChoice>) => {
             originalParentTypeLsids.push(...parentTypes.map(parentType => parentType.type.lsid).toArray());
         });
+
+        // filter out additional parent alias columns that's already added
+        const additionalParentTypeSchemaQueryKeys = [];
+        additionalParentTypes?.forEach(parentType => {
+            additionalParentTypeSchemaQueryKeys.push(
+                parentType.schemaName.toLowerCase() + '|' + parentType.queryName.toLowerCase()
```

Comment:
is there a reason not to just use the `parentType.toString()` (and then likely calling toLowerCase() after) helper here instead of joining the schema and query name together yourself?

Reply:
done